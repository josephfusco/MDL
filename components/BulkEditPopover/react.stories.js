import React from 'react'
import { storiesOf } from '@storybook/react'
import BulkEditPopover from './react'
import { React as TextField } from '../TextField'
import Readme from './react.README.md'

const styles = {
  width: '40px',
}

storiesOf('BulkEditPopover', module)
  .addParameters({ readme: { sidebar: Readme } })
  .add('example', () => (
    <div style={{ marginLeft: 100, marginTop: 100 }}>
      <BulkEditPopover direction="left" cell={'4\'2"'}>
        <div style={styles}>
          <TextField onChange={() => {}} value="" variant="light" removeMargin="true" />
        </div>
      </BulkEditPopover>
    </div>
  ))
