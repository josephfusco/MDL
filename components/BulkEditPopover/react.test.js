import React from 'react'
import {
  render, fireEvent, cleanup, waitForElement,
} from '@testing-library/react'
import BulkEditPopover from './BulkEditPopover'

describe('BulkEditPopover', () => {
  it('should render without crashing', () => {
    expect(render(<BulkEditPopover />)).toMatchSnapshot()
  })
})
