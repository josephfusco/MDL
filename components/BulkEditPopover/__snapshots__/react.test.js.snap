// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableBulkEditPopover rendering with defaults should render correctly 1`] = `
<th
  className="popover-wrapper popover-wrapper--left"
  style={
    Object {
      "position": "absolute",
    }
  }
>
  <div
    className="popover popover--left"
  >
    field
  </div>
  <div
    className="popover-wrapper__cell"
  >
    4.2
  </div>
</th>
`;
